packages/react-core/src/components/TreeView/examples/TreeView.md

```diff
@@ -11,1277 +11,64 @@ import { FolderIcon, FolderOpenIcon, EllipsisVIcon, ClipboardIcon, HamburgerIcon
-                  },
-                  {
-                    name: 'Loading App 3',
-                    id: 'example3-LoadApp3',
-                    checkProps: { checked: false }
-                  }
-                ]
-              }
-            ]
-          }
-        ],
-        defaultExpanded: true
-      },
-      {
-        name: 'Cost management',
-        id: 'example3-Cost',
-        checkProps: { 'aria-label': 'cost-check', checked: false },
-        children: [
-          {
-            name: 'Application 3',
-            id: 'example3-App3',
-            checkProps: { 'aria-label': 'app-3-check', checked: false },
-            children: [
-              {
-                name: 'Settings',
-                id: 'example3-App3Settings',
-                checkProps: { 'aria-label': 'app-3-settings-check', checked: false }
-              },
-              {
-                name: 'Current',
-                id: 'example3-App3Current',
-                checkProps: { 'aria-label': 'app-3-current-check', checked: false }
-              }
-            ]
-          }
-        ]
-      },
-      {
-        name: 'Sources',
-        id: 'example3-Sources',
-        checkProps: { 'aria-label': 'sources-check', checked: false },
-        children: [
-          {
-            name: 'Application 4',
-            id: 'example3-App4',
-            checkProps: { 'aria-label': 'app-4-check', checked: false },
-            children: [
-              {
-                name: 'Settings',
-                id: 'example3-App4Settings',
-                checkProps: { 'aria-label': 'app-4-settings-check', checked: false }
-              }
-            ]
-          }
-        ]
-      },
-      {
-        name: 'Really really really long folder name that overflows the container it is in',
-        id: 'example3-Long',
-        checkProps: { 'aria-label': 'long-check', checked: false },
-        children: [
-          { name: 'Application 5', id: 'example3-App5', checkProps: { 'aria-label': 'app-5-check', checked: false } }
-        ]
-      }
-    ];
-
-    this.state = { checkedItems: [] };
-
-    this.onCheck = (evt, treeViewItem) => {
-      const checked = evt.target.checked;
-      console.log(checked);
-
-      const checkedItemTree = this.options
-        .map(opt => Object.assign({}, opt))
-        .filter(item => this.filterItems(item, treeViewItem));
-      const flatCheckedItems = this.flattenTree(checkedItemTree);
-      console.log('flat', flatCheckedItems);
-
-      this.setState(
-        prevState => ({
-          checkedItems: checked
-            ? prevState.checkedItems.concat(
-                flatCheckedItems.filter(item => !prevState.checkedItems.some(i => i.id === item.id))
-              )
-            : prevState.checkedItems.filter(item => !flatCheckedItems.some(i => i.id === item.id))
-        }),
-        () => {
-          console.log('Checked items: ', this.state.checkedItems);
```

Comment:
I'm curious why you removed these console logs.

Reply:
It doesn't let me commit with console logs

Reply:
Even if you add an `eslint-disable-next-line no-console` linter disabling comment before it?